src/styles/card.scss

```diff
@@ -0,0 +1,98 @@
+.card {
+  box-sizing: border-box;
+
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px;
+  border-style: solid;
+  border-radius: 5px;
+  border-color: $card-border-color;
+
+  &__image {
+    margin-bottom: 36px;
+    margin-right: 3px;
+    margin-left: 3px;
```

Comment:
do we can use just `margin` instead? https://developer.mozilla.org/ru/docs/Web/CSS/margin